crates/ruma-macros/src/api/request/incoming.rs

```diff
@@ -56,7 +56,7 @@ impl Request {
 
             let parse = quote! {
                 let request_query: RequestQuery =
-                    #serde_html_form::from_str(&request.uri().query().unwrap_or(""))?;
+                    #serde_html_form::from_str(request.uri().query().unwrap_or(""))?;
```

Comment:
This is actually not necessary for the edition update, but it is just something that I noticed.